src/sentry/rules/conditions/event_frequency.py

```diff
@@ -75,7 +75,7 @@ class EventFrequencyForm(forms.Form):
     )
 
     def clean(self) -> dict[str, Any] | None:
-        cleaned_data: dict[str, Any] = super().clean()
+        cleaned_data: dict[str, Any] = super().clean() or {}
```

Comment:
I don't think this is the correct refactor here -- should probably be:

```python
cleaned_data = super().clean()
if cleaned_data is None:
    return None
```

and drop the inline annotation